src/screens/Wallets/components/Header.tsx

```diff
@@ -65,20 +63,22 @@ export function HomeHeader(): JSX.Element {
     const res = (await Cache.getItem(CacheKey.LastNotificationTimestamp)) || 0;
```

Comment:
`(await Cache.getItem(CacheKey.LastNotificationTimestamp))` would lead res to be parsed as {}, resulting in TS error in line 65.

Reply:
would this resolve the issue?

const res = (await Cache.getItem(CacheKey.LastNotificationTimestamp)) as
      | Date
      | 0;

Reply:
I think `as number` should be instead of `Date`.
```
const res = ((await Cache.getItem(CacheKey.LastNotificationTimestamp)) as number) || 0;
```